rust/azure_iot_operations_services/src/azure_device_registry/models/device.rs

```diff
@@ -86,9 +86,9 @@ pub struct DiscoveredDeviceEndpoints {
 /// Represents the outbound endpoints of a device in the Azure Device Registry service.
 #[derive(Debug, Clone)]
 pub struct OutboundEndpoints {
-    /// The 'assigned' Field.
+    /// Device messaging endpoint model.
```

Comment:
any difference in description between assigned and unassigned ?

Reply:
I thought this one was pretty weird too! But the new descriptions from the DTDL (what I have here) still at least felt better than what we had before, and I'm not sure that I know enough to come up with something else